src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeEventInfo.cs

```diff
@@ -53,8 +54,7 @@ internal override bool CacheEquals(object? o)
             return
                 o is RuntimeEventInfo m &&
                 m.m_token == m_token &&
-                RuntimeTypeHandle.GetModule(m_declaringType).Equals(
-                    RuntimeTypeHandle.GetModule(m.m_declaringType));
+                m_declaringType.Equals(m.m_declaringType);
```

Comment:
Was `Type.Equals` choosen over Module equality since it's faster? Or is it a correctness thing?

Reply:
I believe because it is faster, also assuming correctness also satisfied as .Net Core support only one module types are unique within a module